tests/functional/pages/features/routing/answer_count/group0-block.page.js

```diff
@@ -0,0 +1,17 @@
+// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
+const QuestionPage = require('../../../surveys/question.page');
+
+class Group0BlockPage extends QuestionPage {
```

Comment:
If the schema had more of context then these page models may look less generic

Reply:
So the review point here is to rename the `Group[0-2]` to something else?